src/database/database.js

```diff
@@ -9,13 +9,28 @@ export const database = {
     const CONNECTION_STRING = env.get('CONNECTION_STRING')
     this.client = await db.postgres.connect(CONNECTION_STRING)
 
+    this.client.once('error', async () => {
+      this.client.end()
+      await this.reconnect()
+    })
+
     for (const callback of connectCallbacks) {
       callback()
     }
 
     return this
   },
 
+  async reconnect() {
+    const timeout = env.get('DATABASE_RECONNECTION_TIMEOUT', '3000')
```

Comment:
Is there any reason why we use `'3000'` instead of just `3000`?

Reply:
I kept the habit from typescript, where `env.get('DATABASE_RECONNECTION_TIMEOUT', 3000)` fails because `env.get('DATABASE_RECONNECTION_TIMEOUT')` returns a string but the default value is a number

So, it's so the type is consistent. We can change it if you think it's best!

Reply:
I'm just asking because we have an int in all the fallbacks values which are expecting a number. I think that it would be better to use `3000` to be aligned with the rest but is up to you 😄.